common/models/change.js

```diff
@@ -96,7 +99,9 @@ module.exports = function(Change) {
     });
 
     async.parallel(tasks, function(err) {
-      if (err) return callback(err);
+      if (err) {
+        callback(err);
+      }
```

Comment:
No need to change this, the `return` is for the anonymous function passed to `async.parallel`.

Please revert this block.